src/duration.rs

```diff
@@ -742,38 +742,76 @@ impl Duration {
 // region: trait impls
 /// The format returned by this implementation is not stable and must not be relied upon.
 ///
+/// By default this produces an exact, verbose printout of the duration.
```

Comment:
I think it would be better to describe the default as "precise", rather than "verbose".

Reply:
"precise" would be redundant with "exact", I think. I'm just trying to get across the idea of "unless you want the exact duration down to the nanosecond, it's probably more verbose than you want".

Reply:
"exact, full-precsision" maybe?

Reply:
"exact, full-precision" works for me.